SU-HSDO/suhumsci
# SHS-5771: Make the Paragraph Drag and Drop option more visible to Site Editors

# READY FOR REVIEW

## Summary
* Adds a patch and config to move the Paragraph drag and drop action from the paragraph three dot menu to the paragraph toolbar.
* Patch was created by @codechefmarc for the Paragraphs module (https://www.drupal.org/project/paragraphs/issues/3470569) and adds the following option to the widget settings form for paragraphs:

![Screenshot 2024-08-27 at 3 41 05 PM](https://github.com/user-attachments/assets/cc0e0890-3d63-459a-9bc1-2ec56cd19039)

## Need Review By (Date)
2024-09-11

## Urgency
low

## Steps to Test
1. Login to the site
2. Add a Flexible page
3. Verify that in the Main Region, there is a "Drag and drop" button that appears on the table header of the paragraph edit form.

![Screenshot 2024-08-27 at 3 22 08 PM](https://github.com/user-attachments/assets/60761a02-757f-480b-8d94-c0ea299b5721)

4. Add a Hero Image into the Full-Width Region
5. Verify that the full-width region also gets a Drag and Drop button in the table header.

![Screenshot 2024-08-27 at 3 22 55 PM](https://github.com/user-attachments/assets/2f084e2f-b436-48d2-9f24-ac9e90f6daf9)

7. Add a Private page
8. Verify that the components paragraph also gets a Drag and drop button in the table header
9. Click on the Drag & drop button and verify that drag and drop still works

![Screenshot 2024-08-27 at 3 25 14 PM](https://github.com/user-attachments/assets/207c559c-231f-4277-843f-e72b2adb0b85)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208164376448509
